docs/API_reference.md

```diff
@@ -562,6 +564,36 @@ There are currently two supported output formats: 'json' and 'dataframe'.
 The 'json' format simply returns the raw response, without page information and with multiple pages combined into a single data structure.  
 The 'dataframe' format transforms that data structure into a pandas DataFrame, which contains subparts in json, that can be split further by the user application.
 
+#### Plenty BI related data <a name='get-bi-section'></a>
+
+##### plenty_api_get_contacts <a name='get-bi'></a>
```

Comment:
~~You forgot to change the title of the section.~~
I would prefer if you squash your commits on a PR, it is quite frustrating to review when the changes are split into multiple commits.
And I also would prefer if new features are added with the least amount of commits in order to easily find them in the git history.